src/helpers.js

```diff
@@ -3,30 +3,50 @@
 import FS from 'sb-fs'
 import Path from 'path'
 import semver from 'semver'
-import { exec } from 'sb-exec'
+import { BufferedProcess } from 'atom'
 import type { Dependency } from './types'
 
 let shownStorageInfo = false
 const VALID_TICKS = new Set(['✓', 'done'])
 const VALIDATION_REGEXP = /(?:Installing|Moving) (.*?) to .* (.*)/
 
+export function exec(command: string, parameters: Array<string>): Promise<{ stdout: string, stderr: string }> {
```

Comment:
Any reason this is being `export`ed? Also, why not mark it as `async`? Even though it makes no functional difference here, it is nice syntactic sugar to say to anyone reading it "Hey, this function will need to have an `await`/`.then()`.